### ISSUE TITLE
Inconsistency between design summary and TODO in Component Implementation 2

### ISSUE THREAD

**Issue by markmccrum** [fprime-community/fprime-workshop-led-blinker#29]
The design summary states that the number of LED transitions is to be reported as telemetry. In the code excerpt for `Led::run_handler` there is a TODO suggesting that an event should be used instead.

```
                // TODO: Add an event to report the number of LED transitions (this->transitions)
```
It probably makes more sense for it to be telemetry.

**Comment by LeStarch** [comment]
Fixed by #30

**Comment by kevin-f-ortega** [comment]
And https://github.com/fprime-community/fprime-workshop-led-blinker/pull/33
